modules/helfi_paragraphs_news_list/helfi_paragraphs_news_list.module

```diff
@@ -148,9 +148,8 @@ function helfi_paragraphs_news_list_entity_bundle_info_alter(array &$bundles) :
  * Implements hook_preprocess_paragraph().
  */
 function helfi_paragraphs_news_list_preprocess_paragraph__news_list(&$variables) {
-  $language_id = \Drupal::languageManager()
-    ->getCurrentLanguage(LanguageInterface::TYPE_CONTENT)
-    ->getId();
+  $paragraph = $variables['paragraph'];
+  $language_id = \Drupal::languageManager()->getCurrentLanguage()->getId();
```

Comment:
Was this `$language_id` change intentional?

Reply:
It was not. Reverted to previous version